common/schedule.c

```diff
@@ -147,6 +147,9 @@ static void context_switch_out(struct vcpu *vcpu)
 	if (vcpu == NULL)
 		return;
 
+	/* cancel event(int, gp, nmi and exception) injection */
```

Comment:
Maybe the vcpu in context_switch_out is not the current vmcs structure. In such case the exec_vmwrite won't work as expected.
It will be better that the exec_vmptrld is called for the safety.

Reply:
Hi Yakui,
Thanks for the comments.
Now, we only have CPU partition feature (no CPU sharing yet). So from performance perspective, we tries best to avoid loading VMCS. There is comments in line 168 explaining it and what should be done if we want to support CPU sharing.